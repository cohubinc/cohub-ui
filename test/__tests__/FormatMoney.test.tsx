import React from "react";
import { mount } from "enzyme";

import { FormatMoney } from "dist";

it("renders FormatMoney without crashing", async () => {
  const wrapper = mount(<FormatMoney value={0} />);

  expect(wrapper).toMatchSnapshot();
});
