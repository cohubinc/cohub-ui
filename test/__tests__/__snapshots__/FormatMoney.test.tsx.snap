// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FormatMoney without crashing 1`] = `
<FormatMoney
  extendedPrecision={true}
  value={0}
>
  <span>
    <NumberFormat
      allowEmptyFormatting={false}
      allowLeadingZeros={false}
      allowNegative={true}
      decimalScale={2}
      decimalSeparator="."
      displayType="text"
      fixedDecimalScale={true}
      getInputRef={[Function]}
      isAllowed={[Function]}
      isNumericString={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onMouseUp={[Function]}
      onValueChange={[Function]}
      prefix="$"
      suffix=""
      thousandSeparator={true}
      thousandsGroupStyle="thousand"
      type="text"
      value={0}
    >
      <span>
        $0.00
      </span>
    </NumberFormat>
  </span>
</FormatMoney>
`;
